pkgs/servers/sql/postgresql/ext/pgrouting.nix

```
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, postgresql, perl, cmake, boost, gmp, cgal, mpfr }:
+
+stdenv.mkDerivation rec {
+  pname = "pgrouting";
+  version = "2.6.2";
+
+  nativeBuildInputs = [ cmake perl ];
+  buildInputs = [ postgresql boost gmp cgal mpfr ];
+
+  src = fetchFromGitHub {
+    owner  = "pgRouting";
+    repo   = pname;
+    rev    = "refs/tags/v${version}";
```

Using `ref/tags/` is not really necessary since [tags have precedence over branches](https://git-scm.com/docs/git-rev-parse.html#_specifying_revisions)
```suggestion
    rev    = "v${version}";
```

---

Done!